gravitee-apim-console-webui/src/portal/top-bar/portal-top-bar.component.html

```diff
@@ -15,5 +15,46 @@
     limitations under the License.
 
 -->
-<portal-header title="Top Bar" subtitle="Customize the Developer Portal top bar" />
-<router-outlet></router-outlet>
+<portal-header title="Manage your top bar sections" subtitle="Manage and edit your Developer Portal pages with Markdown" />
+
+<main class="page-layout">
+  <section class="panel sections-panel">
+    <header class="panel-header sections-panel__header">
+      <h3>Sections</h3>
```

Comment:
According to [figma](https://www.figma.com/design/eAS6p0kITPnEI0VPbmATi3/Developer-Portal?node-id=25137-2935&t=WF533tolcSA0rdpt-1), Sections should be renamed to Items
```suggestion
      <h3>Items</h3>